main.py

```
@@ -272,21 +271,21 @@
-        
+  
         Args:
             Dictionary containing information about the comic
             including filepath and database id.
         """
         cont = ReadingController(comic)
         cont.read_comic()
 
-    def print_hi(self) -> None:
-        print("Hi")
+    def print_hi(self, comic_dict) -> None:
```

Ruff (0.11.9)

Missing type annotation for function argument `comic_dict`

(ANN001)